tools/interop_matrix/ruby_v101_docker_image_patch.patch

```diff
@@ -0,0 +1,17 @@
+--- Dockerfile	2017-11-15 00:18:20.834713531 +0000
```

Comment:
2 or more files can be patched by a single .patch file, so no need to have 2 patches -> please merge into one.

Reply:
I stopped using `diff` and `patch` and started using `git diff` and `git apply` in the latest update. I think this should be most flexible.